css/index.css

```diff
@@ -1,3 +1,478 @@
+		flex-shrink: 1;
+		flex-grow:1;
+		}
+
+    .main .navbar{
+	flex-direction: column-reverse;
+	position: fixed;
+	justify-content: space-between;
+	margin: 0;
+    background: white;
+    z-index: 1;
+	top: 0;
+	width: 80vw;
+	flex-shrink: 1;
+	flex-grow: 1;
+	
+	}
+	.main .navbar nav {
+		height: 20px;
+		flex-shrink: 1;
+		flex-grow: 1;
+		background: white;
+	}
+	
+    .main .navbar nav a{
+		display: flex;
+		flex-direction: row;
+		margin-right: 5vw;
+		margin-bottom: 75px;
+		
+        justify-content: space-around;
+        background: white;
+	}
+	.main .navbar nav a:nth-of-type(6){
+		margin-right:0;
+	}
+	
+    .main .navbar .logo {
+        margin-bottom: 40px;
+	}
+	
+    .main .wowmuchlarge .headimg {
+        display: none;
+	}
+	
+    .main .wowmuchlarge {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        margin: 0;
+		margin-bottom: 100px;
+		padding: 0;
+    }
+    .main .wowmuchlarge .muchtext {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        text-align: center;
+    }
+    .main .wowmuchlarge .muchtext #getitdone{
+    height: 30px;
+	width: 150px;
+    border: 1px solid black;
+    text-align: center;
+    padding-top: 5px;
+	}
+	.contact {
+		align-content: flex-start;
+	}
+	.middle-img{
+		height: 200px;
+		width: 100%;
+	}
+	footer {
+		margin-top: 100px;
+		}
+}
+
+
+/* mobile view */
+@media only screen and  (max-width: 500px){
+	
+	.main {
+		width: 80vw;
+		padding: 10px;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+    }
+    .main .navbar{
+
+		flex-direction: column-reverse;
+		position: relative;
+		height: 40vh;
+		width: 80vw;
+		background: white;
+		z-index: 1;
+		top: 0;
+		margin-bottom: 5px;
+	}
+
+	.main .navbar nav {
+			display: flex;
+			width: 80vw;
+			padding:0;
+			margin: 0;
+
+			align-items: stretch;
+			background: white;
+			flex-direction: column;
+			justify-content: flex-start;
+		}
+		
+		.main .navbar nav a {
+			margin: 0;
+			padding-top: 10px;
+			padding-bottom: 10px;	
+			height: 40px;
+			background: white;
+			border-top: 1px solid gray;
+			border-bottom: 1px solid gray;
+		}
+		
+		.main .navbar .logo {
+			margin-bottom: 10px;
+		}
+		
+		.middle-img {
+			width: 1vh;
+		}
+	.main .wowmuchlarge {
+		position: relative;
+		height: 100px;
+		top: 5px;
+	}
+	 .container-above {
+		display: flex;
+		flex-direction: column;
+		justify-content: stretch;
+		width: 80vw;
+		margin-top: 40px;
+	}
+
+	.container-below {
+		display: flex;
+		flex-direction: column;
+		justify-content: stretch;
+		width: 80vw;
+		margin-top: 40px;
+	}
+	.container-below .services,.container-below .product, .container-below .vision, .container-above .features, .container-above .about {
+		flex-direction: column;
+		align-items: stretch;
+		width: 80vw;
+	}
+
+	.middle-img{
+		height: 100px;
+		width: 100%;
+	}
+	footer {
+	margin-top: 100px;
+	}
+}
```

Comment:
Hey, Chris just gonna give you some quick feedback. Pro tip: when coding try and keep your code clean no worries if not if you are using vscode you can press Alt + Shift + F and it will do it for you just make sure you are clicked into the file you want made pretty